client/me/billing-history/transactions-header.jsx

```diff
@@ -41,17 +47,20 @@ class TransactionsHeader extends React.Component {
 		this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
 	};
 
-	getDatePopoverItemClickHandler( analyticsEvent, filter ) {
+	getDatePopoverItemClickHandler( analyticsEvent, date ) {
 		return () => {
 			this.recordClickEvent( 'Date Popover Item: ' + analyticsEvent );
-			this.handlePickerSelection( filter );
+			const { transactionType } = this.props;
```

Comment:
Nit: I've noticed we usually tend to define variables/constants at the top of methods.